examples/simple-grep/README.md

```diff
@@ -181,11 +181,11 @@ app = do
     formattedPrinter "Starting grepping 🔥" Colourista.white
     file <- liftIO $ TIO.readFile filePath
 
-    let fileName = "file name: " `T.append` (last $ T.split (== '/') $ T.pack filePath)
+    let fileNameMessage = "file name: " `T.append` (last $ T.split (== '/') $ T.pack filePath)
```

Comment:
Hmm, the last commit messages says that it uses `takeFileName` to get rid of the `last` function usage and remove Stan warning but I still see `last` here 🤔 

Maybe something was lost during conflict resolution?